wiki/Chat_Console/id.md

```diff
@@ -88,9 +88,9 @@ Tekan tombol angka atau menekan salah satu bar untuk:-
 | `/msg [pengguna] [pesan]`     | Mengirimkan sebuah pesan pribadi kepada `[pengguna]`.                                                                                                                                                                              | `/msg Amigo` Saya sedang sakit.    | (Di tab Amigo) John: Saya sedang sakit.                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | `/np`                         | Menampilkan lagu yang sedang kamu dengarkan atau yang kamu mainkan pada obrolan                                                                                                                                                  | `/np`                              | (If playing) * John is playing [Peter Lambert - osu! tutorial\[Gameplay Basics\]](https://osu.ppy.sh/b/22538) [Game Modes](/wiki/FAQ#alternative-game-modes) \~[Relax/Autopilot](/wiki/Game_Modifiers#special-mods)\~ +[Difficulty increasing mod](/wiki/Game_Modifiers#difficulty-increasing-mods) -[Difficulty decreasing mod](/wiki/Game_Modifiers#difficulty-reduction-mods)/[SpunOut](/wiki/Game_Modifiers#spun-out) [xK](Game_Modifiers#xk) (If doing nothing, based on jukebox) * John is listening to [Peter Lambert - osu! tutorial](https://osu.ppy.sh/b/22538) (If spectating or Autoplay) * John is watching [Peter Lambert - osu! tutorial\[Gameplay basics\]](https://osu.ppy.sh/b/22538) [Game Modes](/wiki/FAQ#alternative-game-modes) \~[Relax/Autopilot](/wiki/Game_Modifiers#special-mods)\~ +[Difficulty increasing mod](/wiki/Game_Modifiers#difficulty-increasing-mods) -[Difficulty decreasing mod](/wiki/Game_Modifiers#difficulty-reduction-mods)/[SpunOut](/wiki/Game_Modifiers#spun-out) [xK](Game_Modifiers#xk) [Autoplay](/wiki/Game_Modifiers#auto) |
```

Comment:
Make this line shorter until only just:

>|`/np`| Menampilkan lagu yang sedang kamu dengarkan atau yang kamu mainkan pada obrolan.| `/np`| (If playing) * John is playing [Peter Lambert - osu! tutorial \[Gameplay Basics\]](https://osu.ppy.sh/b/22538) | 

as it's related to [this](https://github.com/ppy/osu-wiki/commit/47dc2ed40e8f54a4515cef25ecaaadd60601dc98#diff-80480f4ce6c10cf415cc281f6f05df97) change.